--- a/src/input_source.h	2017-04-09 22:11:27.000000000 +0200
+++ b/src/input_source.h	2017-04-17 00:00:50.721697310 +0200
@@ -31,7 +31,7 @@
 	public:
 		Source() = default;
 
-		virtual ~Source() = default;
+		virtual ~Source() {};
 
 		/**
 		 * Called once each frame to update pressed_buttons.
@@ -53,7 +53,7 @@
 	class UiSource : public Source {
 	public:
 		UiSource() = default;
-		~UiSource() override = default;
+		~UiSource() override {};
 
 		void Update() override;
 
