diff -Naur a/src/output.h b/src/output.h
--- a/src/output.h	2016-09-11 02:10:08.000000000 +0200
+++ b/src/output.h	2016-09-25 23:34:48.894615767 +0200
@@ -107,7 +107,7 @@
 	 *
 	 * @param fmt formatted error to display.
 	 */
-	[[noreturn]] void Error(const char* fmt, ...);
+	void Error(const char* fmt, ...) __attribute__((noreturn));
 
 	/**
 	 * Display an error message and closes the player
@@ -115,7 +115,7 @@
 	 *
 	 * @param err error to display.
 	 */
-	[[noreturn]] void ErrorStr(std::string const& err);
+	void ErrorStr(std::string const& err) __attribute__((noreturn));
 
 	/**
 	 * Prints a debug message to the console.
