diff --git a/src/game_character.h b/src/game_character.h
index bee52b0f3..210bf0010 100644
--- a/src/game_character.h
+++ b/src/game_character.h
@@ -383,13 +383,13 @@ class Game_Character {
 	 * @return turning max_stop_count for the given frequency
 	 * @param freq input movement frequency
 	 */
-	constexpr int GetMaxStopCountForTurn(int freq);
+	static constexpr int GetMaxStopCountForTurn(int freq);
 
 	/**
 	 * @return waiting max_stop_count for the given frequency
 	 * @param freq input movement frequency
 	 */
-	constexpr int GetMaxStopCountForWait(int freq);
+	static constexpr int GetMaxStopCountForWait(int freq);
 
 	/**
 	 * Sets the max_stop_count
