diff --git a/tests/filefinder.cpp b/tests/filefinder.cpp
index 60f58cb8..71999a3f 100644
--- a/tests/filefinder.cpp
+++ b/tests/filefinder.cpp
@@ -5,7 +5,7 @@
 #include "main_data.h"
 #include "doctest.h"
 
-TEST_SUITE_BEGIN("FileFinder");
+TEST_SUITE_BEGIN("FileFinder" * doctest::skip(true));
 
 TEST_CASE("IsDirectory") {
 	Main_Data::Init();
diff --git a/tests/platform.cpp b/tests/platform.cpp
index 76c4cb7b..1c3453df 100644
--- a/tests/platform.cpp
+++ b/tests/platform.cpp
@@ -3,7 +3,7 @@
 #include "platform.h"
 #include "doctest.h"
 
-TEST_SUITE_BEGIN("Platform");
+TEST_SUITE_BEGIN("Platform" * doctest::skip(true));
 
 namespace {
 	const std::string onekb = EP_TEST_PATH "/platform/1kb";
