--- src/sdl_ui.cpp.orig	2017-04-09 22:11:27.000000000 +0200
+++ src/sdl_ui.cpp	2017-04-16 21:00:26.830656211 +0200
@@ -839,9 +839,12 @@
 		return;
 
 	int amount = evnt.wheel.y;
+
+#if SDL_VERSION_ATLEAST(2,0,4)
 	// translate direction
 	if (evnt.wheel.direction == SDL_MOUSEWHEEL_FLIPPED)
 		amount *= -1;
+#endif
 
 	keys[Input::Keys::MOUSE_SCROLLUP] = amount > 0;
 	keys[Input::Keys::MOUSE_SCROLLDOWN] = amount < 0;
